docs/operating-scylla/procedures/cluster-management/remove-seed.rst

```diff
@@ -5,7 +5,7 @@ Remove a Seed Node from Seed List
 This procedure describes how to remove a seed node from the seed list.
 
 .. note::
-   The seed concept in gossip has been removed. Starting with ScyllaDB Open Source 4.3 and ScyllaDB Enterprise 2021.1, a seed node 
+   The seed concept in gossip has been removed. A seed node 
```

Comment:
I'm still not sure why you decided to keep this entire text but leave out the name of the ancient version where this was done. I think it's useful information to know that this was done in 2021, not recently.